protocol/src/main/java/org/geysermc/mcprotocollib/protocol/data/game/entity/object/Direction.java

```diff
@@ -10,16 +10,14 @@ public enum Direction implements ObjectData {
     WEST(2),
     EAST(3);
 
+    public static final Direction[] VALUES = values();
+    private static final Direction[] HORIZONTAL_VALUES = {NORTH, SOUTH, WEST, EAST};
     @Getter
     private final int horizontalIndex;
-
```

Comment:
why remove the space between fields and the constructor? we're not doing that elsewhere